presto-parquet/src/main/java/com/facebook/presto/parquet/reader/MetadataReader.java

```diff
@@ -124,11 +125,14 @@ public static ParquetMetadata readFooter(FSDataInputStream inputStream, Path fil
                             .map(value -> value.toLowerCase(Locale.ENGLISH))
                             .toArray(String[]::new);
                     ColumnPath columnPath = ColumnPath.get(path);
-                    PrimitiveTypeName primitiveTypeName = messageType.getType(columnPath.toArray()).asPrimitiveType().getPrimitiveTypeName();
+                    final org.apache.parquet.schema.Type type = messageType.getType(columnPath.toArray());
```

Comment:
ditto: we don't use `final` modifiers in local variables.

Reply:
Done